src/index.js

```diff
@@ -110,6 +110,13 @@ const options = {
         description: "Python 3"
       }
     ]
+  },
+  __isInTest: {
```

Comment:
Does this show up in `prettier --help`?

Reply:
Oh womp yes 🤦 Seems like using `global` is the best option then? I can change it back :)

Reply:
`process.env.NODE_ENV === 'test'`?

Reply:
Yup, that works!